commands/poll.js

```diff
@@ -0,0 +1,151 @@
+const Discord = require('discord.js');
+const emojiRegex = require('emoji-regex');
+
+const symbols = require('../symbols');
+const functions = require('../functions');
+
+function generateTestResults(options) {
+	op = {...options};
+	for (let i = 0; i < Object.keys(op).length; i++) {
+		op[Object.keys(op)[i]] = Math.floor(Math.random() * 100);
+	}
+	return op;
+}
+function generateResults(options) {
+	op = {...options};
+	for (let i = 0; i < Object.keys(op).length; i++) {
+		op[Object.keys(op)[i]] = 0;
+	}
+	return op;
+}
+
+module.exports = {
+	name: 'poll',
+	description: 'Run a poll in a specified channel using reactions',
+	execute(client, message, args) {
+		let poll_options = {};
+		if (args.length > 0) {
```

Comment:
I think it might be nice to restrict permissions to who can invoke it. Maybe include
a "Exec role" into the config and verify the message up against that role. I'm
not 100% sure why others should be allowed to poll?

Reply:
That's a good idea, I am restricting the endpoll command to the poll creator, but if we have any other commands that we might not want anyone else running then I'll have to implement this. For now, I think it's ok that anyone can start a poll.